src/components/Registration/NatureOfBusiness.vue

```diff
@@ -5,13 +5,13 @@
     :class="{ 'invalid-section': showErrors && !isFormValid }"
   >
     <v-row>
-      <v-col cols="12" sm="2">
+      <v-col cols="12" sm="3">
```

Comment:
I believe "small" is the correct breakpoint, but this makes the label row a little larger until it finally wraps. (Subtle UI tweak.)